src/editors/containers/TextEditor/components/SourceCodeModal/messages.js

```diff
@@ -0,0 +1,18 @@
+export const messages = {
```

Comment:
The names here should be edited to be related to the new modal. `uploadButtonLabel` should be removed. `saveButtonLabel` should say "Save". `titleLabel` should say "Source Code".